Source/Engine/Core/Config/LayersAndTagsSettings.cs

```diff
@@ -30,6 +31,29 @@ public static string[] GetCurrentLayers()
             return GetCurrentLayers(out int _);
         }
 
+        /// <summary>
+        /// The layers names.
+        /// </summary>
+        [EditorOrder(10), EditorDisplay("Terrain Layers", EditorDisplayAttribute.InlineStyle), Collection(CanResize = false, Display = CollectionAttribute.DisplayType.Inline)]
+        public string[] TerrainLayers = Enumerable.Repeat(string.Empty, 8).ToArray();
+
+        /// <summary>
+        /// Gets the current terrain layer names. Returns "Layer" + index for layers without a name.
+        /// </summary>
+        /// <returns>The layer names.</returns>
+        public static string[] GetCurrentTerrainLayers()
+        {
+            string[] layerNames = GameSettings.Load<LayersAndTagsSettings>().TerrainLayers.ToArray();
```

Comment:
Use `FLAX_TESTS` macro to return empty array so Tests will pass (some C# tests run without C++ engine backend thus they fail on compilation).